python/packaging/frontend_sdist/setup.py

```diff
@@ -38,6 +39,22 @@ def run_pip_command(args, call_func):
     except subprocess.CalledProcessError:
         return call_func([os.path.join(sys.exec_prefix, "bin", "pip")] + args)
 
+# cherry-pick information from `packaging.markers.default_environment()` needed to find the right wheel
+# https://github.com/pypa/packaging/blob/23.1/src/packaging/markers.py#L175-L190
+if sys.platform == "linux":
+    platform_marker = "manylinux_2_17"
```

Comment:
Doesn't seem like these are actually being used anywhere

Reply:
yeah they're unused variables (see commit linked), I can clean them up if you like the idea of this PR